G/glm/build_tarballs.jl

```diff
@@ -0,0 +1,35 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "glm"
```

Comment:
The upstream project seems to style it GLM, all capital:
```suggestion
name = "GLM"
```
Also, please note that you'll have to rename the directory accordingly

Reply:
i don't mind one way or another, but there's already a GLM package (https://juliastats.org/GLM.jl/stable/) and the header file names and namespace are lowercased (unlike, for instance, with SDL). that's why i went for lowercase.

but let me know what you think given that and i'll make any needed changes.

Reply:
If you are worried about name conflicts, anyway in the registry we can't have two packages with exact same name apart from case: that wouldn't work on case-insensitive file systems (such as those typically used on Windows and macOS) :slightly_smiling_face: All JLL packages will have `_jll` appended to the name, so the conflict isn't a concern.  The problem would be if here we had another package called `GLM_jll` already: in that case any case wouldn't be acceptable, because of the registry issue above.

Reply:
that's very interesting, but i still like lowercase for the package name since their git repo, header files, install directory names, and C++ namespace are all lowercase, as distinct from packages like SDL where they are uppercase. thus i think it's clearer to keep the name lowercase, rather than risk confusion with the uppercase GLM.

...on the other hand, it's probably referred to in uppercase in official docs.

idk if there's a "right" answer, both sides have a case. :stuck_out_tongue_closed_eyes:

what do you think? do you want me to go ahead change it?

Reply:
i looked at the package naming guidelines and didn't find an obvious answer, but there is point 4 "Err on the side of clarity, even if clarity seems long-winded to you." What would you think of "OpenGLMath" as an alternative to "glm" or "GLM"?

Reply:
Maybe that's better, yes

Reply:
ok, that's what i'll do